documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -6,22 +6,9 @@
 [id="about-cold-warm-migration_{context}"]
 = About cold and warm migration
 
-{project-short} supports cold migration from {rhv-full} ({rhv-short}) and warm migration from VMware vSphere and from {rhv-short}.
+{project-short} supports cold migration from {rhv-full} ({rhv-short}), {ocp-short}, {osp}, VMware vSphere, and Open Virtial Application files that originate from or are compatible with VMware vSphere.
```

Comment:
```suggestion
{project-short} supports cold migration from VMware vSphere, {rhv-full} ({rhv-short}), {osp}, and {virt}
```
1. I'd keep the order in which the source providers were introduce, it's kind of aligned with the importance
2. like before, replace OCP with OpenShift Virtualization (it would be even better to have something like a "remote OpenShift Virtualization cluster")
3. OVA cannot start so cold/warm migration is irrelevant for OVA - instead of fixing the typo (Virtial), I'd drop that part

Reply:
Just to be clear --- OVA belongs in the list in master.adoc (line 24, etc.) , but not here, correct?

Reply:
> Just to be clear --- OVA belongs in the list in master.adoc (line 24, etc.) , but not here, correct?

correct